src/networking.c

```diff
@@ -2726,7 +2726,7 @@ void readQueryFromClient(connection *conn) {
          *
          * For unauthenticated clients, the query buffer cannot exceed 1MB at most. */
         (c->mstate.argv_len_sums + sdslen(c->querybuf) > server.client_max_querybuf_len ||
-         (c->mstate.argv_len_sums + sdslen(c->querybuf) > 1024*1024*1024 && authRequired(c)))) {
```

Comment:
LOL..
when i looked at it, i was so busy fighting the urge to suggest `1<<20` that i didn't pay attention to count how many `1024` instances we have.